pkg/lib/aws/errors.go

```diff
@@ -131,3 +132,10 @@ func ErrorReadCredentials() error {
 		Message: "unable to read AWS credentials from credentials file",
 	})
 }
+
+func ErrorECRExtractingCredentials() error {
+	return errors.WithStack(&errors.Error{
+		Kind:    ErrECRExtractingCredentials,
+		Message: "can't extract ECR credentials",
```

Comment:
nit: "unable to extract ECR credentials"

Reply:
Done!